index.bs

```diff
@@ -312,6 +312,15 @@ noise suppression using Recurrent Neural Network such as [[RNNoise]] for
 suppressing background dynamic noise like baby cry or dog barking to improve 
 audio experiences in video conferences.
 
+### Detecting fake video ### {#usecase-detecting-fake-video}
+
+A user is exposed to realistic fake videos generated by ‘deepfake’ on the web. 
+The fake video can swap the speaker’s face into the president’s face to incite 
+a user politically or to manipulate user’s opinion. The deepfake detection 
+applications such as [[FaceForensics++]] analyze the videos and protect a user against 
+the fake videos or images. When she watches a fake video on the web, the 
+detection application alert her the fraud video in real-time.
```

Comment:
```suggestion
detection application alerts her of the fraud video in real-time.
```

Reply:
Thanks for review and comments!
And I hope to contribute on ethical and societal issues for WebNN.